backend/src/helpers/sql.ts

```diff
@@ -23,12 +26,12 @@ interface EventData {
   type?: string;
 }
 
-interface UserData {
-  firstName?: string;
-  lastName?: string;
-  phone?: string;
-  email?: string;
-}
+// interface UserData {
+//   firstName?: string;
+//   lastName?: string;
+//   phone?: string;
+//   email?: string;
+// }
```

Comment:
Can these commented out lines be removed?

Reply:
Done